src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -127,8 +316,7 @@ public List<Point> Locations
         // without breaking replication guides
 
         [IsVisibleInDynamoLibrary(false)]
-        [AllowRankReduction]
-        public static AdaptiveComponent[] ByParametersOnFace(double[][][] uvs, ElementFaceReference faceReference, FamilyType familyType)
```

Comment:
To maintain binary compatibility, we can't really remove this method. Can you please update this method`s implementation based on the new method. Also mark this method as [SuppressImportInVM] so this method is not available in DS Engine.